Java_fibonacci.java

```diff
@@ -0,0 +1,27 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class Java_fibonacci {
```

Comment:
Codacy found an issue: [The class name 'Java_fibonacci' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288781)